plugins/stronghold/guest-js/index.ts

```diff
@@ -298,7 +298,14 @@ export class Store {
       snapshotPath: this.path,
       client: this.client,
       key: toBytesDto(key),
-    }).then((v) => Uint8Array.from(v));
+    }).then((v) => {
+         if (v !== null || undefined) {
+          Uint8Array.from(v);
+         }
+         else {
+          null
+        }
+      })
```

Comment:
You're not returning anything here so it will always be null (or undefined?) no matter if the value exists or not.

Reply:
Better ?

Reply:
same issue still. That only work in Rust, in javascript/typescript you'll have to use the `return` keyword here